emperor/pandas.py

```diff
@@ -0,0 +1,116 @@
+r"""
+Pandas Interface for Emperor
+============================
+
+This module provides a simple interface to visualize a Pandas DataFrame using
+the Emperor.
+
+.. currentmodule:: emperor.pandas
+
+Functions
+---------
+.. autosummary::
+    :toctree: generated/
+
+    scatterplot
+"""
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, emperor development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE.md, distributed with this software.
+# ----------------------------------------------------------------------------
+from __future__ import division
+
+import numpy as np
+import pandas as pd
+
+from emperor.core import Emperor
+from skbio import OrdinationResults
+
+
+def scatterplot(df, x=None, y=None, z=None, remote=True):
+    """Create an Emperor scatter plot from a Pandas DataFrame
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        Pandas DataFrame with the data to display
+    x, y, z : str
+        Column names in `df`, these will represent the order of the axes in the
+        final visualization.
```

Comment:
Can we add a comment here about why these parameters for `x`, `y`, `z` are optional?  I'm assuming that they will take the first 3 columns right?